zerver/lib/export.py

```diff
@@ -160,6 +160,7 @@ class MessagePartial(TypedDict):
     "zerver_realmreactivationstatus",
     "zerver_realmuserdefault",
     "zerver_recipient",
+    "zerver_savedreply",
```

Comment:
I expect this needs `import_realm.py` code as well, to renumber the `UserProfile` and future `Realm` foreign keys. You can test it by creating a `savedreply` object in the early part of `test_import_export.py` and having it diff the before/after database state.

Reply:
So, is this correct?:

```py
if "zerver_savedreply" in data:
        re_map_foreign_keys(data, "zerver_savedreply", "user_profile", related_table="user_profile")
        re_map_foreign_keys(data, "zerver_savedreply", "realm", related_table="realm")
        update_model_ids(SavedReply, data, "savedreply")
        bulk_import_model(data, SavedReply)
```